services/apps/portal/src/application-review/application-review.controller.ts

```diff
@@ -156,9 +158,21 @@ export class ApplicationReviewController {
       userLocalGovernment.isFirstNation,
     );
 
+    const validationResult =
+      await this.applicationValidatorService.validateApplication(application);
+
+    if (!validationResult.application) {
+      throw new BaseServiceException(
+        `Invalid application found during LG Submission ${application.fileNumber}`,
```

Comment:
Do I get it right that later this message will be dynamic based on the application type?

Reply:
This is more a safeguard, this error should never happen as the application was validated when the Applicant submitted it. So unless something went wrong then the application should still be valid after the LG review is added.